addons/website_event/models/event_event.py

```diff
@@ -102,7 +102,7 @@ def _compute_is_participating(self):
           * logged as visitor: check partner or visitor are linked to a
             registration;
         """
-        current_visitor = self.env['website.visitor']._get_visitor_from_request(force_create=False)
+        current_visitor = self.env['website.visitor']._get_visitor_from_request()
```

Comment:
Unnecessary diff. Moreover giving value for this optional parameter clearly states we don't want to create the visitor, to indicate we know what we are doing (compared to "we used default value because lol").